src/components/create-curriculum-component/CreateCurriculumComponent.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Grid, Paper, Checkbox, TextField } from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import SearchIcon from '@material-ui/icons/Search';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+
+const icon = <CheckBoxOutlineBlankIcon fontSize="small" />;
+const checkedIcon = <CheckBoxIcon fontSize="small" />;
+
+interface IData {
+    skill: string;
+}
+
+
+
+export function CreateCurriculumComponent() {
+
+    return (
+        <div>
+            <br />
+            <Grid container justify="center">
+                <Paper component="form" >
+
+                    <Autocomplete
+                        popupIcon={<SearchIcon />}
+                        placeholder="test"
+                        multiple
+                        id="checkboxes-tags-demo"
+                        options={skillList}
```

Comment:
If I am reading this right, this should be all skills from the api

Reply:
It will be, but we wanted to get this working before we got started on that.